protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -75,91 +80,196 @@ async fn test_movement_client_should_publish_package() -> Result<(), anyhow::Err
+                                Amount(AssetType::Moveth(amount))
+                        ).await.expect("Failed to lock bridge transfer");
+
+                let details = movement_client
+                        .get_bridge_transfer_details(BridgeTransferId(bridge_transfer_id)).await
+                        .expect("Failed to get bridge transfer details")
+                        .expect("Expected to find bridge transfer details, but got None");
+
+                assert_eq!(details.bridge_transfer_id.0, bridge_transfer_id);
+                assert_eq!(details.hash_lock.0, hash_lock);
+                assert_eq!(
+                        &details.initiator_address.0 .0[32 - initiator.len()..],
+                        &initiator,
+                        "Initiator address does not match"
+                );
+                assert_eq!(details.recipient_address.0, recipient.0.to_vec());
+                assert_eq!(details.amount.0, AssetType::Moveth(amount));
+                assert_eq!(details.state, 1, "Bridge transfer is supposed to be locked but it's not.");
+
+                movement_client
+                        .complete_bridge_transfer(
+                                BridgeTransferId(bridge_transfer_id),
+                                HashLockPreImage(b"secret".to_vec())
+                        ).await
+                        .expect("Failed to complete bridge transfer");
+
+                let details = movement_client
+                        .get_bridge_transfer_details(BridgeTransferId(bridge_transfer_id)).await
+                        .expect("Failed to get bridge transfer details")
+                        .expect("Expected to find bridge transfer details, but got None");
+
+                assert_eq!(details.bridge_transfer_id.0, bridge_transfer_id);
+                assert_eq!(details.hash_lock.0, hash_lock);
+                assert_eq!(
+                        &details.initiator_address.0 .0[32 - initiator.len()..],
+                        &initiator,
+                        "Initiator address does not match"
+                );
+                assert_eq!(details.recipient_address.0, recipient.0.to_vec());
+                assert_eq!(details.amount.0, AssetType::Moveth(amount));
+                assert_eq!(details.state, 2, "Bridge transfer is supposed to be completed but it's not.");
+
+                Ok(())
+        }.await;
+
+        if let Err(e) = child.kill().await {
+                eprintln!("Failed to kill child process: {:?}", e);
+        }
+
+        test_result
+}
 
 
-		let faucet_client = faucet_client.write().unwrap();
 
-		faucet_client
-		.fund(movement_client.address(), 100_000_000)
-		.await?;
 
-		// Check the balance of movement_client after funding
-		let balance = coin_client
-		.get_account_balance(&movement_client.address())
-		.await?;
+#[tokio::test]
+async fn test_movement_client_should_successfully_call_lock_and_abort() -> Result<(), anyhow::Error> {
```

Comment:
This test currently fails with the provided command. From what I'm reading, could this be an issue to hard coded environment variables?
```
024-09-03T22:14:34.734579Z ERROR movement_bridge: Failed to get transfer item: Api(AptosErrorResponse { error: AptosError { message: "Table Item not found by Table handle(0xe513451190640deb971ca1c98037382f284fb240358541c91eb672d8d607206), Table key(\"0x30303030303030303030303030303030303030303030307472616e7366657231\") and Ledger version(63)", error_code: TableItemNotFound, vm_error_code: None }, state: Some(State { chain_id: 4, epoch: 2, version: 63, timestamp_usecs: 1725401673714174, oldest_ledger_version: 0, oldest_block_height: 0, block_height: 28, cursor: None }), status_code: 404 })
thread 'test_movement_client_should_successfully_call_lock_and_abort' panicked at protocol-units/bridge/integration-tests/tests/eth_movement.rs:212:14:
Failed to get bridge transfer state: ViewSerializationError
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
FAILED

failures:

failures:
    test_movement_client_should_successfully_call_lock_and_abort

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 11 filtered out; finished in 33.38s
```

Reply:
I think you're using an old version of the branch, because I don't have "Failed to get bridge transfer state" anymore in the most up-to-date version. Mind doing a `git pull` and trying again?

Reply:
```
test test_movement_client_should_successfully_call_lock_and_abort ... STDERR: 
STDERR: 
STDERR: Readiness endpoint: http://127.0.0.1:8070/
STDERR: 
STDERR: Transaction stream is starting, please wait...
STDERR: Faucet is starting, please wait...
STDERR: Node API is starting, please wait...
STDERR: 
STDOUT: Completed generating configuration:
STDOUT:         Log file: "/Users/primata/movement/.movement/testnet/validator.log"
STDOUT:         Test dir: "/Users/primata/movement/.movement/testnet"
STDOUT:         Aptos root key path: "/Users/primata/movement/.movement/testnet/mint.key"
STDOUT:         Waypoint: 0:a83030a61d1156ba131fa924efbdfb7e312d3ae64145ad1820699754759d57bb
STDOUT:         ChainId: 4
STDOUT:         REST API endpoint: http://127.0.0.1:8080
STDOUT:         Metrics endpoint: http://127.0.0.1:9101/metrics
STDOUT:         Aptosnet fullnode network endpoint: /ip4/0.0.0.0/tcp/6181
STDOUT:         Indexer gRPC node stream endpoint: 127.0.0.1:50051
STDOUT: 
STDOUT: Aptos is running, press ctrl-c to exit
STDOUT: 
STDERR: 
STDERR: One of the services failed to start up, running shutdown steps...
STDERR: Ran shutdown steps
STDOUT: Unexpected error: 
STDOUT: One of the services crashed on startup:
STDOUT: Ok(
STDOUT:     Err(
STDOUT:         Error {
STDOUT:             context: "Service ended with an error",
STDOUT:             source: Os {
STDOUT:                 code: 48,
STDOUT:                 kind: AddrInUse,
STDOUT:                 message: "Address already in use",
STDOUT:             },
STDOUT:         },
STDOUT:     ),
STDOUT: )
STDOUT: Please check the logs in /Users/primata/movement/.movement/testnet
thread 'test_movement_client_should_successfully_call_lock_and_abort' panicked at /Users/primata/movement/protocol-units/bridge/chains/movement/src/lib.rs:173:33:
Failed to receive setup completion signal: RecvError(())
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
FAILED
```

Reply:
The "Address already in use" indicates that you already have a process running on the port. 

Run `ps aux | grep movement` to get the PID of the process. 

Then run `kill -9 <pid>`

Then try again and you shouldn't get the "Address already in use" error.

Reply:
To confirm, the tests are working locally for me.